src/components/Navigation.js

```diff
@@ -0,0 +1,56 @@
+import { NavLink } from "react-router-dom";
+
+import styled from "styled-components";
+
+function Navigation() {
+  return (
+    <NavUl>
```

Comment:
I think would be enough to name it just `Nav` ord `NavWrapper`. `NavUl` is not mandatory because I can see, that there is a list in there.